analyzer/src/funTest/kotlin/managers/PipFunTest.kt

```diff
@@ -116,4 +117,14 @@ class PipFunTest : WordSpec({
     }
 })
 
-private fun createPip() = Pip("PIP", USER_DIR, DEFAULT_ANALYZER_CONFIGURATION, DEFAULT_REPOSITORY_CONFIGURATION)
+private fun createPip(pythonVersion: String = "3.10") =
```

Comment:
Shouldn't this use `OPTION_PYTHON_VERSION_DEFAULT`?

Reply:
I want the tests to be decoupled from the implementation in this regard, there will be a follow-up which cleans up the tests a bit, e.g. by running some test for linux and windows, and while at it I will consider if it makes sense to couple some tests to the default option value.